src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationItem.cs

```diff
@@ -262,72 +186,43 @@ public void AddPermalink(Uri permalink)
         {
             if (permalink == null)
             {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("permalink");
+                throw new ArgumentNullException(nameof(permalink));
             }
-            this.Id = permalink.AbsoluteUri;
-            this.Links.Add(SyndicationLink.CreateAlternateLink(permalink));
-        }
 
-        public virtual SyndicationItem Clone()
-        {
-            return new SyndicationItem(this);
+            Id = permalink.AbsoluteUri;
+            Links.Add(SyndicationLink.CreateAlternateLink(permalink));
         }
 
-        public Atom10ItemFormatter GetAtom10Formatter()
-        {
-            return new Atom10ItemFormatter(this);
-        }
+        public virtual SyndicationItem Clone() => new SyndicationItem(this);
 
-        public Rss20ItemFormatter GetRss20Formatter()
-        {
-            return GetRss20Formatter(true);
-        }
+        public Atom10ItemFormatter GetAtom10Formatter() => new Atom10ItemFormatter(this);
+
+        public Rss20ItemFormatter GetRss20Formatter() => GetRss20Formatter(true);
 
```

Comment:
Inconsistent conversion of one-line methods to delegate-based method as the other overload of GetRss20Formatter is a one line method wasn't converted.